libcontainer/configs/cgroup_unix.go

```diff
@@ -16,6 +16,9 @@ type Cgroup struct {
 	// name of parent cgroup or slice
 	Parent string `json:"parent"`
 
+	// CgroupsPath specifies the path to cgroups that are created and/or joined by the container.
+	CgroupsPath string `json:"cgroupsPath"`
```

Comment:
If we want to avoid adding an additional field, we could overload Name and treat it as cgroupsPath if it is absolute. That way we don't need to have Parent, Name as well as CgroupsPath. WDYT?